packages/tofu/package.json

```
@@ -26,5 +26,5 @@
   "bugs": {
     "url": "https://github.com/LavaMoat/lavamoat/issues"
   },
-  "homepage": "https://github.com/LavaMoat/lavamoat#readme"
+  "homepage": "https://github.com/LavaMoat/LavaMoat/blob/main/packages/viz/README.md"
```

this should be tofu, not viz
```suggestion
  "homepage": "https://github.com/LavaMoat/LavaMoat/blob/main/packages/tofu/README.md"
```